libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -24,7 +26,8 @@ export class BookSearchComponent implements OnInit {
 
   constructor(
     private readonly store: Store,
-    private readonly fb: FormBuilder
+    private readonly fb: FormBuilder,
+    private _snackBar: MatSnackBar
```

Comment:
We don't typically apply underscores to private variables. See previously defined variables.

Reply:
updated